pages/linux/bzip2.md

```diff
@@ -0,0 +1,15 @@
+# bzip2
+
+> A block-sorting file compressor, alias - bunzip2.
```

Comment:
This is not a well-formed sentence. Please replace the part after the comma with a proper sentence (or even omit that here, and mention it in the description of the decompression example)